test/integration-tests.md

```diff
@@ -0,0 +1,171 @@
+## List of current integration tests
```

Comment:
How is this list to be maintained as we add/remove integrations tests?

Reply:
+1 on an automated solution.

Otherwise, I feel like it'll likely get out of date.
Is there an ticket that's tracks the automation side of it?